site/_apps/preact-hacker-news.txt

```diff
@@ -14,11 +14,11 @@ hosting: 'now.sh'
 authors:
```

Comment:
Out of interest did this implementation start using Preact CLI or was it from scratch? Just asking as it it was the former would be good to capture that. No worries if not however :)

Reply:
Thanks for the confirm!